src/jni_core/src/jvm_runtime.rs

```diff
@@ -0,0 +1,291 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use core::option::Option::Some;
+use core::result::Result::{Err, Ok};
+use std::ffi::c_void;
+use std::fs;
+use std::path::Path;
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::LazyLock;
+
+use jni::strings::JNIString;
+use jni::{InitArgsBuilder, JNIVersion, JavaVM, NativeMethod};
+use risingwave_common::util::resource_util::memory::total_memory_available_bytes;
+
+use crate::run_this_func_to_get_valid_ptr_from_java_binding;
+
+pub static JVM: LazyLock<Option<JavaVM>> = LazyLock::new(|| {
+    let libs_path = if let Ok(libs_path) = std::env::var("CONNECTOR_LIBS_PATH") {
+        libs_path
+    } else if std::env::var("ENABLE_BUILD_RW_CONNECTOR").is_ok() {
+        // If ENABLE_BUILD_RW_CONNECTOR is set which means it is started from risedev with a
+        // connector.
+        ".risingwave/bin/connector-node/libs/".to_string()
+    } else {
+        return None;
+    };
```

Comment:
I haven't come up with a better idea of how to pass the connector java libs path to compute nodes or meta nodes. 
Is an environment variable acceptable? If yes, after this PR, our Kubernetes need to specify this variable for compute and meta, otherwise, we can't initialize the JVM properly.

Reply:
Of course, if we don't want to use the functionality provided by the connector, we can just ignore `CONNECTOR_LIBS_PATH`.

Reply:
Looks like the default Java class path is the current working directory. Can we directly put the `jar` in the same directory with the RisingWave binary for the cloud deployment?

Reply:
Some imaginary ideas: We can even directly include the release `jar` in the RisingWave binary (with `include_bytes`)! 😄 Then users only need to download a single artifact to deploy or test locally.

Reply:
> Looks like the default Java class path is the current working directory. Can we directly put the `jar` in the same directory with the RisingWave binary for the cloud deployment?

I think for the cloud deployment, our image has already included the jars we need. We even know the path in that image, but I am not sure whether we should hard code this path.